CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.0 (TBD)
+* Added `onInitNgModel` event
+* Replace `keyup` listener for `input`
```

Comment:
It might be worthwhile mentioning this applies to the `modelEvents`?

Reply:
Good call. Done!